src/store/profile/types.ts

```diff
@@ -1,8 +1,12 @@
+import { ICountryTemplates, IRequisite, ITemplate } from '@uspacy/sdk/lib/models/requisites';
```

Comment:
```suggestion
import { IErrorsAxiosResponse } from '@uspacy/sdk/lib/models/errors';
import { ICountryTemplates, IRequisite, ITemplate } from '@uspacy/sdk/lib/models/requisites';
```
У нас есть интерфейс для ошибок в axios